server/test/unit/server/db/model/test_resources.py

```diff
@@ -1,55 +1,21 @@
 """
 This module contains tests for the pulp.server.db.model.resources module.
 """
-import mock
-import uuid
+import unittest
 
-from ....base import ResourceReservationTests
 from pulp.server.db.model import resources
 
 
-class TestReservedResource(ResourceReservationTests):
+class TestReservedResource(unittest.TestCase):
 
-    def test_unique_indices(self):
-        self.assertEqual(resources.ReservedResource.unique_indices, tuple())
+    def test_attributes(self):
+        rr = resources.ReservedResource('my-task', 'some-worker', 'some-resource')
```

Comment:
Since these are all class attributes, you shouldn't actually instantiate the class. Instead make assertions on the class itself. I think the tests @barnabycourt are a really good example. See:  https://github.com/pulp/pulp/pull/1796/files#diff-7b4c3232751ba4e87aa0dd34e5a81ba4R23